owlkettle/widgets.nim

```diff
@@ -243,13 +252,16 @@ renderable Box of BaseWidget:
               vAlign: AlignFill.}:
     ## Adds a child to the Box.
     ## When expand is true, the child grows to fill up the remaining space in the Box.
-    ## The hAlign and vAlign properties allow you to set the alignment of the child
-    ## within its allocated area.
+    ## The hAlign and vAlign properties allow you to set the horizontal and vertical 
+    ## alignment of the child within its allocated area. They may be one of `AlignFill`, 
+    ## `AlignStart`, `AlignEnd` or `AlignCenter`.
+    ## 
+    ## Note: **Any** widgets contained in a Box-Widget get access the `expand` adder, to control their behaviour inside of the Box!
```

Comment:
I would remove this note. I think how adders work should be described somewhere in general. (Also: `expand` would be called a property here, since it is a parameter to the `Box.add` adder. So e.g. `{.name: "value".}` → Property; `{.name.}` → Adder; `{.name, prop: value.}` → Adder with property)

Reply:
I'll try to come up with a decent text-addition to internals.md and remove the note. I'll need to find some proper time to hammer that one out so I'll have to postpone that to "later", but I'll see what I can do!

Reply:
Done